components/Settings.js

```diff
@@ -360,6 +381,12 @@ class Settings extends React.PureComponent {
         </div>
         <div className="settings-settings">
           <WindowPointer fromLeft="15px" />
+          <Presets
+            show={showPresets}
+            toggle={this.togglePresets}
+            presets={PRESETS}
```

Comment:
@raboid let's add `PRESETS` to `state` in `Editor` and then pass that value in as a prop into `Settings`. That way we can support adding/removing presets easily.